contrib/devtools/symbol-check.py

```diff
@@ -72,6 +72,25 @@
     },
 }
 
+ELF_ABIS: Dict[lief.ELF.ARCH, Dict[lief.ENDIANNESS, List[int]]] = {
+    lief.ELF.ARCH.x86_64: {
+        lief.ENDIANNESS.LITTLE: [3,2,0],
+    },
+    lief.ELF.ARCH.ARM: {
+        lief.ENDIANNESS.LITTLE: [3,2,0],
+    },
+    lief.ELF.ARCH.AARCH64: {
+        lief.ENDIANNESS.LITTLE: [3,7,0],
+    },
+    lief.ELF.ARCH.PPC64: {
+        lief.ENDIANNESS.LITTLE: [3,10,0],
```

Comment:
I tested all these versions manually first by reading them with `readelf --notes bitcoin-cli` and then running this script against the 24.0.1 release binaries. All the others matched, but the power pc little endian version gave me `3.2.0`. Against which binaries did you check these?

Reply:
This is against master (binaries produced with a Guix build). The symbol/security scripts are (only) expected to work with the version of the source they are shipped with. In this case, we are using a newer version of glibc to produce the binaries in master, hence the difference. Think I also alluded to this in a comment above.

Reply:
Ah right, all the better that we catch these now. I found a recent guix build and it checks out there.